psychopy/core.py

```diff
@@ -114,9 +115,9 @@ def shellCall(shellCmd, stdin='', stderr=False, env=None, encoding='utf-8'):
     when running Python 2.7.
 
     """
-    if env is None:
-        env = dict()
-
+    if encoding == None:
```

Comment:
We should probably use `is` instead of `==` for this singleton comparison. :)

Reply:
True